features/step_definitions/oauth.rb

```diff
@@ -1,18 +1,26 @@
-Given /^the #{QUOTED} oauth CRD is restored after scenario$/ do |name|
+# The resource is a "customresource" (CR), not "customresourcedefinition" (CRD). Correcting it and make existing use not broken
+Given /^the #{QUOTED} oauth CR(?: D)? is restored after scenario$/ do |name|
```

Comment:
Perhaps we need to update the scenarios which use this step as well. Or else people will just reference to those existing "wrong" examples.

Reply:
+1, and the regexp here may not match the existing scenarios.

```
irb(main):013:0> /oauth CR(?: D)? is/.match("oauth CRD is")
=> nil
```